app/controllers/AccessibilityController.scala

```diff
@@ -32,7 +32,7 @@ class AccessibilityController @Inject() (
 
   extends FrontendBaseController(cc) {
 
-  def getAccessibilityStatement(): Action[AnyContent] = Action { implicit request =>
-    Ok(views.accessibilityStatement())
+  def getAccessibilityStatement(): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
I don't think this controller needs to be changed; there's no need for a `Future` here or for the action to be `.async`